test/definitions/ohi/linux/cassandra-debian.json

```diff
@@ -0,0 +1,63 @@
+{
+  "global_tags": {
+    "owning_team": "OpenSource",
+    "Environment": "development",
+    "Department": "Product",
+    "Product": "Virtuoso"
+  },
+
+  "resources": [{
+    "id": "host1",
+    "display_name": "Debian10InfraCassandraInstallHost",
+    "provider": "aws",
+    "type": "ec2",
+    "size": "t3.nano",
+    "user_name": "admin",
+    "ami_name": "SupportedImages debian-10-amd64-*"
+  }],
+
+  "services": [{
+    "id": "cassandra1",
+    "destinations": ["host1"],
+    "source_repository": "https://github.com/newrelic/open-install-library.git",
+    "deploy_script_path": "test/deploy/linux/nginx/install/debian/roles",
+    "port": 80
+  }],
+
+  "instrumentations": {
+    "resources": [{
+        "id": "nr_infra_cassandra",
+        "resource_ids": ["host1"],
+        "provider": "newrelic",
+        "source_repository": "https://github.com/newrelic/open-install-library.git",
+        "deploy_script_path": "test/deploy/linux/newrelic-cli/install-recipe/roles",
+        "params": {
+          "recipe_content_url": [
+            "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/infra-agent/debian.yml",
+            "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/cassandra/debian.yml"
+          ]
+        }
+      },
+      {
+        "id": "nr_infra_is_having_data",
+        "resource": ["host1"],
+        "provider": "newrelic",
+        "source_repository": "https://github.com/newrelic/open-install-library.git",
+        "deploy_script_path": "test/deploy/assertions/recipe-is-valid/roles",
+        "params": {
+          "nrql_query": ""
```

Comment:
check the most recent nginx test definition, we should use a params element to use the recipe validationNrql https://github.com/newrelic/open-install-library/blob/main/test/definitions/ohi/linux/nginx-debian.json#L49